guides/Building Your Bot/CreatingPointsSystems.md

```diff
@@ -33,7 +33,7 @@ module.exports = class extends Monitor {
 		if (!message.guild) return;
 
 		// Update the user's configuration entry by adding 1 to it.
-		await message.author.settings.update('experience', message.author.settings.experience + 1);
+		await message.author.settings.update('experience', message.author.get('experience') + 1);
```

Comment:
Is that suppose to be `message.author.settings.get('experience')`?

Reply:
Whoops, yes, can you update your message to show a suggestion? So I can commit the fix co-authored-by you

Reply:
Done.